dataframes/04-reading-messy-data-into-dataframes.ipynb

```diff
@@ -0,0 +1,502 @@
+{
```

Comment:
<p>This will sneakily read the first file to infer the schema if you don't pass in <code>meta=...</code> . Depending on how expensive computing the first partition is this could be fine, or it might be bad.</p><p> </p><p>Here the <code>meta</code>  would be something like:</p><p> </p><pre class="ql-syntax" spellcheck="false">meta = [("id", int), ("name", str), ("x", float), ("y", float)]
</pre><p> </p><p>A dict from column name -&gt; dtype also works (e.g. <code>{"id": int, ...}</code>).  This is similar enough to the <code>dtype</code>  kwarg used above that you might share them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dask/dask-examples/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>I originally had the meta in there, will put it back in.</p>